### PR TITLE
PAE-1677: Return empty WB columns when accreditation is null

### PR DESCRIPTION
**Pull request by senthilinfy**
Ticket: [PAE-1677](https://eaflood.atlassian.net/browse/PAE-1677)
- Fix waste record export showing WB column values for operators with cancelled accreditations or reg-only processing types
- Short-circuit `getWasteBalanceClassification` to return null when `accreditation === null`, avoiding incorrect INCLUDED classification
- Render Included in Waste Balance, Waste Balance Exclusion Reason, and Waste Balance Tonnage as empty when classification is null
- Add unit and integration tests covering cancelled accreditation and null classification scenarios

[PAE-1677]: https://eaflood.atlassian.net/browse/PAE-1677?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/waste-records-export/application/stream-csv-export.test.js [hunk 1]
```diff
@@ -585,6 +596,38 @@ describe('streamCsvExport', () => {
...
+      wasteRecordsRepository: {
+        findByRegistration: vi
+          .fn()
+          .mockResolvedValue([reprocessorReceivedRecord()])
+      }
+    })
+
+    const out = await collect(streamCsvExport(deps))
+    const cells = out[1].trim().split(',')
+    expect(cells[5]).toBe('No') // Accredited
```
Comment (GraemeF) [inline src/waste-records-export/application/stream-csv-export.test.js]: could these column index numbers be named?

**Review by GraemeF — APPROVED** [review]
